aws_topology/tox.ini

```diff
@@ -0,0 +1,29 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py27
+envlist =
+    {py27,py36}-aws_topology
```

Comment:
If you make this:
```
{py27,py3}-aws_topology
```
It will pick whatever python3 version is running. That or add `py36, py37, py38`